static/sass/components/_hero.scss

```diff
@@ -25,3 +25,54 @@
     }
   }
 }
+
+// Hero Images
+// ===========
+
+
+.hero {
+  max-height: 35vh;
```

Comment:
lol, but with the message at top and everything, you can't even see what the page is about if I make them any larger.